src/style/app.module.css

```diff
@@ -379,6 +382,7 @@
   min-width: 150px;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`